clients/egui/src/account/modals/help.rs

```diff
@@ -0,0 +1,57 @@
+use eframe::egui;
+use egui_extras::{Size, TableBuilder};
+
+#[derive(Default)]
+pub struct HelpModal;
+
+impl HelpModal {
+    pub fn show(&mut self, ui: &mut egui::Ui) {
+        TableBuilder::new(ui)
+            .cell_layout(egui::Layout::left_to_right().with_cross_align(egui::Align::Center))
+            .striped(true)
+            .column(Size::exact(100.0))
+            .column(Size::remainder().at_least(60.0))
+            .header(30.0, |mut header| {
+                header.col(|ui| {
+                    ui.label(egui::RichText::new("Shortcut").strong().underline());
+                });
+                header.col(|ui| {
+                    ui.label(egui::RichText::new("Action").strong().underline());
+                });
+            })
+            .body(|mut body| {
+                for (k, v) in [
+                    ("Ctrl-N", "Open the New File prompt"),
+                    ("Ctrl-Space, Ctrl-L", "Open the search prompt"),
```

Comment:
The first column seems to be too small to show both shortcuts.

Reply:
Good catch, https://github.com/lockbook/lockbook/pull/1284/commits/97a4f76cd15d77b92d4e64a03f944fdd54eefef2